trailblazer/server/utils.py

```diff
@@ -39,3 +40,12 @@ def stringify_timestamps(data: dict) -> dict[str, str]:
 
 def parse_job_create_request(request: Request) -> CreateJobRequest:
     return CreateJobRequest.model_validate(request.json)
+
+
+def parse_summaries_request(request: Request) -> SummariesRequest:
+    query_params = {
+        key[:-2]: request.args.getlist(key)
+        for key in request.args.keys()
+        if key_has_list_of_values(key)
+    }
```

Comment:
I find it slightly confusing with the indexes, but maybe it just needs getting used to. What would be the difference from doing a model_validate on the request.args?

Reply:
The `key[:-2]` discards the leading `[]` before the name of the query parameter. It is a standard to add a leading `[]` before a list query param. I moved it into the pydantic model instead